setup/migrate.py

```diff
@@ -99,7 +99,7 @@ def migration_8(env):
 	# Delete DKIM keys. We had generated 1024-bit DKIM keys.
 	# By deleting the key file we'll automatically generate
 	# a new key, which will be 2048 bits.
-	os.unlink(os.path.join(env['STORAGE_ROOT'], 'mail/dkim/mail.private'))
+	os.unlink(os.path.join(env['STORAGE_ROOT'], 'mail/dkim/' + env['DKIM_SELECTOR'] + '.private'))
```

Comment:
Not needed. Because this is a migration, the version that is migrated from must be using "mail", not DKIM_SELECTOR

Reply:
ok, what if someone tried to migrate from e.g. `miab` to `miab2`? Better if I retain this DKIM_SELECTOR and additionally unlink `mail.private`?

Reply:
This migration code will not be run for new versions of MIAB, so it can only know about mail.private. That is indeed the minimum to be unlinked. It makes no sense to also unlink DKIM_SELECTOR. That old version of MIAB will never have been run with an alternative value of DKIM_SELECTOR, so such a file will not exist.

Reply:
Understood, thanks! Removed changes to `migrate.py`